systers_portal/templates/meetup/snippets/meetup_sidebar.html

```diff
@@ -0,0 +1,6 @@
+<div class="sidebar-module mb40 btn-group-vertical" role="group">
+  <p><a href="{% url 'about_meetup_location' object.slug%}">About</a></p><br>
```

Comment:
Since "New meetup" and "Add member" are better off being links, rather than success buttons, try making about a info button (not success) and placing it under "Meetup Location Actions".